pkg/operator/vspherecontroller/vspherecontroller_test.go

```diff
@@ -1188,7 +1281,7 @@ func TestEscapeQuotesAndBackslashes(t *testing.T) {
 		{
 			name:     "Already escaped",
 			input:    `Pass\"word\\1234\"`,
-			expected: `Pass\\\"word\\\\1234\\\"`,
+			expected: `Pass\"word\\1234\"`,
```

Comment:
This is a change of behavior. User provided password with `\"` and `\\`, yet the operator creates an ini file whose password will have `"` and `\`.